### PR TITLE
Remove duplicate/misplaced api.Element.attachInternals entry

### PR DESCRIPTION
**Pull request by foolip**
We accidentally added both:
https://github.com/mdn/browser-compat-data/pull/8112
https://github.com/mdn/browser-compat-data/pull/11245

### CODE REVIEW

**Comment by foolip** [comment]
MDN changes in https://github.com/mdn/content/pull/6976.

**Review by vinyldarkscratch — APPROVED** [review]
Great catch, thank you!  I'm approving, but will hold off on merging until the content PR is merged.

**Review by sideshowbarker — APPROVED** [review]

**Pull request merged**
